presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -275,6 +275,11 @@ public PlanOptimizers(
                 new LimitPushDown(), // Run the LimitPushDown after flattening set operators to make it easier to do the set flattening
                 new PruneUnreferencedOutputs(),
                 inlineProjections,
+                new IterativeOptimizer(
+                        stats,
+                        statsCalculator,
+                        estimatedExchangesCostCalculator,
+                        columnPruningRules),
```

Comment:
can you add these rules to subsequent `IterativeOptimizer`?

Reply:
Not really, because `TransformExistsApplyToLateralNode` will transform `EXISTS` with either non-default or default aggregation (depending if the former if possible). Once this is done, there is no going back.